core/lib/external_price_api/src/coingecko_api.rs

```diff
@@ -110,3 +132,38 @@ impl CoinGeckoPriceResponse {
             .and_then(|price| price.get(currency))
     }
 }
+mod test {
+    use std::str::FromStr;
+
+    use zksync_config::configs::ExternalPriceApiClientConfig;
+    use zksync_types::Address;
+
+    use super::CoinGeckoPriceAPIClient;
+
+    #[tokio::test]
+    // This is not a proper test, as it requires user input and prints information.
+    // It serves as a quick check to verify if the CoinGecko api_key is working correctly.
```

Comment:
Do I get it right that the expected workflow here is to change `api_key` and `forced_ethereum_erc20_address` to some valid values, then run the test and manually see if it produces some output?
If so, I think it's better to get input via environment variables.

Reply:
Sounds good, I'm trying this:
```rust
  let config_env = ExternalPriceApiClientConfig::from_env().unwrap();
  let api = CoinGeckoPriceAPIClient::new(config_env);
 ```
 
 And i'm compiling the `dev.toml` config:
 
 ```toml
 [external_price_api_client]
source = "coingecko"
base_url = "https://api.coingecko.com"
api_key = "api"
# Since the coingecko api only works for ethereum mainnet and with the dev config we run a reth instance as L1,
# we can force the erc20 address used to get the conversion_rate, the address below is the DAI contract of ethereum mainnet
forced_ethereum_erc20_address = "0x6b175474e89094c44da98b954eedeac495271d0f"
 ```
 
 But it fails with:
 `Result::unwrap()` on an `Err` value: Cannot load config <external_price_api_client>
 
 Seems that i'm not doing it correctly, i didn't found tests reading from the compiled `env` configs. They are set inside [the test](https://github.com/matter-labs/zksync-era/blob/761bda19844fb3935f8a57c47df39010f88ef9dc/core/lib/env_config/src/external_price_api_client.rs#L34). I would appreciate your insights.